core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooLock.java

```diff
@@ -38,9 +40,13 @@
 import org.slf4j.LoggerFactory;
 
 public class ZooLock implements Watcher {
-  private static final Logger log = LoggerFactory.getLogger(ZooLock.class);
+  private static final Logger LOG = LoggerFactory.getLogger(ZooLock.class);
```

Comment:
Why the capitalization of LOG?  I don't care - but would like to be consistent in the code base.

Reply:
Just a [convention](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) that I have used on my previous teams.

Reply:
We have things that check for code format in our build. The way I see it, if it passes all the checks then you're good. I'd rather have some flexibility in variable naming than have to remember all the rules. I'm sure that we can be trusted to have good judgement.

Reply:
In general, static final variables are typically all uppercase. I'm surprised we don't adopt that convention for more of these loggers. I don't think it matters much, though.